src/redux/services/compare/compareResources.ts

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable no-restricted-syntax */
```

Comment:
Is there a reason why we restrict `for-of`? It reads much easier and I would believe our Electron build is modern enough not to need the generator when compiled?